platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -21,112 +21,96 @@ describing the following:
+and applications.
 
 Enterprises adopting platform engineering can expect to achieve the following:
 
 1. Accelerate product development and delivery by intentionally dedicating teams
-   to platform capabilities and thereby reducing cognitive load on product teams
+   to platform services and thereby reducing cognitive load on product teams
 1. Improve reliability and resiliency of products relying on platform
    capabilities by dedicating experts to configure and manage them
 1. Accelerate product development and delivery by reusing and sharing platform
    tools and knowledge across many teams in an enterprise
 1. Reduce risk of security, regulatory and functional issues in products and
    services by governing platform capabilities and the users, tools and processes
    surrounding them
-1. Enable cost-effective and productive use of capabilities from public clouds
+1. Enable cost-effective and productive use of services from public clouds
    and other providers by enabling delegation of implementations to those providers
    while maintaining control over user experience
 
 These benefits accrue in part because just a few platform teams serve many
 product teams, multiplying their impact; in part because platform teams
-consolidate management of common capabilities, facilitating governance; and in
+consolidate management of common functionality, facilitating governance; and in
 part because platform teams emphasize user interfaces and experiences above all
 else.
 
-A team of platform experts not only reduces common work [1] demanded of product
-teams but also optimizes platform capabilities [2] for those products. And
-because a platform offers patterns for many capabilities, knowledge and tools
-for those spread broadly [3] and developers can quickly contribute to other
-teams and products built on the same foundation. Governance and controls [4] can
-also be embedded in a platform's patterns and capabilities. Finally, because
-platform teams corral capability providers and provide consistent experiences
-over their offerings, they enable efficient use of public clouds and service
-providers [5] for foundational but undifferentiated capabilities like databases
-and identity.
+A team of platform experts not only reduces common work \[1\] demanded of product
+teams but also optimizes platform capabilities \[2\] used in those products. And
+because a platform includes typical vetted patterns, knowledge and tools about
+those spread broadly \[3\] and developers can quickly contribute to other teams
+and products built on the same foundation. Governance and controls \[4\] can also
+be embedded in a platform's patterns and capabilities. Finally, because platform
+teams corral providers and provide consistent experiences over their offerings,
+they enable efficient use of public clouds and service providers \[5\] for
+foundational but undifferentiated capabilities like databases and identity.
```

Comment:
I think this is an awesome point, but I had a bit of a hard time understanding the sentence. Might be due to a lot of long sentences together? Took a stab at rewording it if it helps others.

```suggestion
teams but also optimizes platform capabilities \[2\] used in those products. 
In addition, a platform typical includes vetted patterns, knowledge and tools 
that are well suited to the organisation. Therefore, as platform adoption 
increases, so does the spread of those shared ways of working \[3\] which 
allows developers to quickly contribute to other teams and products built on 
the same foundation. A shared foundations also allows governance and controls 
\[4\] to be embedded with lower overhead cost per team. Finally, because 
platform teams corral providers and provide consistent experiences over their 
offerings, they enable efficient use of public clouds and service providers 
\[5\] for foundational but undifferentiated capabilities like databases and 
identity.
```

Reply:
Working on this, agree that it could flow better.